includes/class-sensei-wp-kses.php

```diff
@@ -92,4 +92,48 @@ public static function maybe_sanitize( $content, $allowed_html )
 
 		return $html_security ? self::wp_kses( $content, $allowed_html ) : $content;
 	}
+
+	/**
+	 * Sanitizes data for allowed HTML tags for an array of elements.
+	 *
+	 * @since 1.12.2
+	 *
+	 * @param array $unescaped_data Array of unescaped data.
+	 * @return array Escaped data.
+	 **/
+	public static function wp_kses_post_array( $unescaped_data ) {
```

Comment:
Since `wp_kses_array` includes the HTML elements for `post`, isn't this function equivalent to just calling `wp_kses_array` with no second argument?